From 111eed0251238a9d3f90e76d62f2ac01aeccce48 Mon Sep 17 00:00:00 2001
From: Allan McRae <allan@archlinux.org>
Date: Wed, 13 Mar 2024 11:33:15 +1000
Subject: [PATCH] Fix unused result warnings

Not actually a fix, as the pacman output will still be weird and
we will not gracefully exit, but it does print an error message...

Signed-off-by: Allan McRae <allan@archlinux.org>
---
 src/pacman/util.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/pacman/util.c b/src/pacman/util.c
index 8a94e8caf..ebc210404 100644
--- a/src/pacman/util.c
+++ b/src/pacman/util.c
@@ -428,10 +428,10 @@ static char *concat_list(alpm_list_t *lst, formatfn fn)
 		}
 
 		if(tmp) {
-			asprintf(&output, "%s %s", tmp, str);
+			pm_asprintf(&output, "%s %s", tmp, str);
 			free(tmp);
 		} else {
-			asprintf(&output, "%s", str);
+			pm_asprintf(&output, "%s", str);
 		}
 		tmp = output;
 
@@ -441,7 +441,7 @@ static char *concat_list(alpm_list_t *lst, formatfn fn)
 	}
 
 	if(!output) {
-		asprintf(&output, "%s", "");
+		pm_asprintf(&output, "%s", "");
 	}
 
 	return output;
-- 
GitLab

